stories/2-Card.stories.js

```diff
@@ -130,3 +133,50 @@ export const NoteCard = () => (
     </FlexCenter>
   </Card>
 );
+
+export const ContextCard = () => (
+  <ContextStyle>
+    <Flex>
+      <div className="context-div">
+        <h1>What is onboarding?</h1>
+        <p>abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz</p>
+      </div>
+      <DoneChecking src={done_checking} />
+    </Flex>
+  </ContextStyle>
+);
+
+export const HintCard = () => (
+  <Background style={{height: 1024, padding: "10px"}}>
+    <React.Fragment>
+      <HintStyle style={{minWidth: "15%", width: "15%"}}>
+        <div className="div" style={{padding: "0 0 20px"}}>
+          <h1>Hint</h1>
+          <p>abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz</p>
+        </div>
+        <DoneChecking style={{width: "80%"}} src={done_checking}/>
+      </HintStyle>
+      <HintStyle style={{border: "none", minWidth: "15.5%", width: "15.5%"}}>
+        <PrimaryWithText />
+      </HintStyle>
+    </React.Fragment>
+  </Background>
+);
+
+export const OnboardingCard = () => (
```

Comment:
apologies, I named this card screenshot wrong in the issue. Can  you rename to "PrimaryWithTextCard"?

Reply:
You got it.